# Blocking Principle
A nuisance factor is a factor that probably has some unintended effect on the response.
There are some ways we can control nuisance factors:

| Nusiance variable type | method |
|---|---|
| known and uncontrollable | Blocking |
| known and uncontrollable | Analysis of covariance |
| unknown and uncontrollable | randomization |

### Randomised Complete Block Design (RCBD)
The RCBD can be intepreted as a design to reduce the residual error in an experiment by removing variability due to known and controllable nuisance variables (via blocking).

Here we can extend the ANOVA linear model to the RCBD.

Given $a$ treatments (factor levels) and $b$ blocks, the effects model for the RCBD is:

\begin{equation}
  y_{ij}=\mu+\tau_i+\beta_j+\epsilon_{ij}=\begin{cases}
    i=1,2,...,a \\
    j=1,2,...,b
  \end{cases}
\end{equation}

The relevant (fixed effects) hypotheses are:

\begin{equation}
    H_0: \tau_1 = \tau_2 ... \tau_a = 0
\end{equation}

Total analysis of variance is:
<center>
\begin{equation}
    \sum_{i=1}^a
    \sum_{j=1}^b
    (y_{ij}-\overline{y})^2
    = \sum_{i=1}^a
    \sum_{j=1}^b[(\overline{y}_i-\overline{y}..)+(\overline{y}_j-\overline{y}..) + (\overline{y}_{ij}-\overline{y}_i-\overline{y}_j+\overline{y}..)]^2 \\
    = b\sum_{i=1}^a(\overline{y}_i-\overline{y}..)^2 + a\sum_{j=1}^b(\overline{y}_j-\overline{y}..)^2 \\
    +\sum_{i=1}^a
    \sum_{j=1}^b(\overline{y}_{ij}-\overline{y}_i-\overline{y}_j+\overline{y}..)^2 \\ SS_T=SS_{Treatments} + SS_{Blocks} + SS_E
\end{equation}

<br/>
The degrees of freedom are:

\begin{equation}
    ab-1=a-1+b-1+(a-1)(b-1)
\end{equation}
</center>
<br/>
Summary of ANOVA for the RBCD:

<img src="https://i.ibb.co/DYnzD8y/Screenshot-2021-07-28-at-4-31-02-PM.png" width="600px" />

In [17]:
# Vascular Graft Experiment

vascular.data <- read.csv('vascular.csv')
anova <- aov(Yield ~ Treatment + Block, data=vascular.data)
summary(anova)

            Df Sum Sq Mean Sq F value  Pr(>F)   
Treatment    3  178.2   59.39   8.107 0.00192 **
Block        5  192.2   38.45   5.249 0.00553 **
Residuals   15  109.9    7.33                   
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

The treatment P-Value = 0.00192 < 0.05 and therefore we reject the null hypothesis.
We can conclude that the treatments do have an effect, i.e extrusion pressure affects yield

# Multi-Blocking Principle
For 2 nuisance factors, we can use the latin square design.

### Latin Square Design (LSD)
LSD is a modified version of the RCBD where both columns and rows are unique. In general, a Latin square for $p$
variables or a $p*p$ Latin square, is a sqaure containing $p$ rows and columns. LSD is an effects model and is completely additive, there is no interation between rows, columns and treatments.

The statistical model for a Latin square is:

\begin{equation}
    y_{ijk} = \mu + \alpha_i + \tau_j + \beta_k + \epsilon_{ijk} \, \text{ for } \, i=1,2,...,p; j=1,2,...,p; k=1,2,...,p \\
    y_{ijk} \text{ is the observation in the }i^{th} \text{row and }k^{th} \text{ column for the } j^{th} \text{treatment} \\
    \mu = \text{ overall mean} \\
    \alpha_i \, \text{is the i}^{th} \text{ row effect} \\
    \tau_j \, \text{is the j}^{th} \text{ treatment effect} \\
    \beta_j \, \text{is the k}^{th} \text{ column effect} \\
    \epsilon_{ijk} \, \text{is the random error}
\end{equation}


\begin{equation}
    \text {Partioning the total sum of suqares of the $N=p^2$ observation:} \\
    SS_T=SS_{Rows}+SS_{Columns}+SS_{Treatments}+SS_{E} \\
\end{equation}

\begin{equation}
    \text{Respective degree of freedom}: \\
    p^2 – 1 = (p – 1) + (p – 1) + (p – 1) + (p – 2)(p – 1) \\
\end{equation}
 
 
\begin{equation}
    \text{The statistic, F-ratio, for testing no differences in treatment means is:} \\
    F_0=\frac{MS_{Treatments}}{MS_E} \text{ is distributed as } F_{p-1, (p-2)(p-1)} \text{ under the null hypothesis. }
\end{equation}
\
\
Summary: \
<img src='https://i.ibb.co/LSKj9yC/Screenshot-2021-07-29-at-12-35-35-AM.png' width='600px' />